elements/pfe-sass/maps/_typography.scss

```diff
@@ -1,9 +1,9 @@
 ////
-/// @group Typography
-/// @author starryeyez024
+/// @group Maps
```

Comment:
I'd keep the author note but move it to the mixins rather than in the global section so that if someone has questions, they can reach out to you.

Reply:
Right that's still "global" syntax.  I meant an author note on individual mixins instead.  If you put it on global and someone else adds a mixin, it's going to assign you as the author.